package.json

```diff
@@ -48,9 +48,9 @@
     "webpack-dev-middleware": "^1.6.1"
   },
   "dependencies": {
-    "favicons": "^4.8.3",
+    "favicons": "^4.8.6",
```

Comment:
Is it possible to update favicons to at least `5.0.0` to fix also this issue https://github.com/jantimon/favicons-webpack-plugin/issues/146 ?

I see that was fixed in this PR of favicons library:

* itgalaxy/favicons#235

Maybe is necessary to fix some breaking changes...

Reply:
Probably it would break other things, can we start merging this?